### DISCUSSION TITLE
Duet Basics

### DISCUSSION THREAD

**Discussion by deepak121993** [OpenMined/courses#288] [Course: Foundations of Private Computation]
Hi Team,

I'm not getting options to get access to Duet Basics after Duet Basics setup.
Are these lessons are not opened yet? Or I'm missing something.

Thanks,
Deepak

**Comment by AbinavRavi** [comment]
Please try the quick fix mentioned in [this thread](https://github.com/OpenMined/courses/discussions/272). It worked for me.

**Reply by deepak121993** [reply]
Thanks, Abinav , it worked

**Reply by deepak121993** [reply]
Hi Abhinav,

Getting the below error.
AttributeError: module 'syft' has no attribute 'launch_duet'

Thanks,
Deepak

**Comment by znreza** [comment]
@deepak121993 could you solve your problem? Have you run the cells like below? 

In Data Owner notebook, run 
```
import syft as sy
duet = sy.launch_duet(network_url="http://ec2-18-216-8-163.us-east-2.compute.amazonaws.com:5000/")
```
If it still doesn't work, I would suggest restart the both notebooks and try again. Let us know if it solved your problem.

**Reply by gonzalo-munillag** [reply]
It does not, I faced the same issue and opened a conversation just for [this](https://github.com/OpenMined/courses/discussions/298)
Cheers

**Reply by deepak121993** [reply]
Hi Gonzalo,
There is versioning issue. By default the Syft installable not supporting this feature. Check out the below conversation. 
[https://github.com/OpenMined/courses/discussions/273#discussioncomment-494352]

You need to follow below steps:

1. pip uninstall syft
2. pip install git+https://github.com/OpenMined/PySyft@dev#egg=syft

or what I did, I had hardcoded the version itself as mentioned in comment:

**pip install syft-0.3.0.post0.dev754+g65a4aa120**

This works for me.

Thanks,
Deepak
